pilot/pkg/serviceregistry/kube/controller/serviceexportcache.go

```diff
@@ -121,3 +143,14 @@ func (c disabledServiceExportCache) isExported(types.NamespacedName) bool {
 func (c disabledServiceExportCache) HasSynced() bool {
 	return true
 }
+
+func (c disabledServiceExportCache) ExportedServices() []model.ExportedService {
+	// MCS is disabled - show everything as exported.
+	return []model.ExportedService{
+		{
+			Cluster:   "*",
```

Comment:
We should document the `*` semantic somewhere?

Or maybe just return `nil` here, which would be different than `[]` which means no service disabled. That may be a bit subtle though..

Reply:
Yeah I like `nil` better ... done.